wiki/Beatmaps/es.md

```diff
@@ -0,0 +1,60 @@
+# Beatmaps
+
+Los beatmaps son los niveles a batir en [osu!](/wiki/osu!_glossary), similares a los de los [Juegos de DS] (/wiki/Glossary/#ds-games).Los beatmaps se componen de una canción y varios [hit objects](/wiki/hit_objects] situados en el espacio y el tiempo al ritmo de la música. Pueden tener, como fondo, una imagen, un vídeo o un [storyboard](/wiki/storyboard).
+
+Los beatmaps pueden subirse a la web a través del [BSS](/wiki/BSS) (Beatmap Submission System: sistema de envío de beatmaps) y son sometidos a un proceso de aprobación llamado [modding](/wiki/modding) para mantener los máximos estándares de calidad posibles en términos de jugabilidad.Puedes encontrar los beatmaps en el [buscador de mapas](https://osu.ppy.sh/beatmapsets) en la web de osu!
+
+## Estatus de rankeo
+
+Los mapas rankeados y aprobados requieren la aprobación de dos [Beatmap Nominators (BN)](/wiki/Beatmap_Nominators). Los mapas rankeados y aprobados afectan a las estadísticas de los usuarios cuando son jugados, a diferencia de los mapas [pendientes (pending)](#pending) o [abandonados (graveyarded)](#graveyard) beatmap.
+
+### Beatmaps rankeados
+
+Los beatmaps rankeados (ranked) vienen indicados con este icono: (![Double upwards chevron icon](/wiki/shared/status/ranked.png)) en la pantalla de selección de canciones. Estos beatmaps siguen a rajatabla todos los [criterios de ranking](/wiki/ranking_criteria) para asegurar que tienen unos mínimos de calidad y jugabilidad, así como que la información que se muestra en ellos es correcta.
+
+### Beatmaps aprobados
+
+Los beatmaps aprobados (approved) vienen indicados con este icono: (![Check icon](/wiki/shared/status/approved.png))  en la pantalla de selección de canciones. El estatus de aprobado es utilizado para canciones que superan los 5 minutos de longitud, que son conocidos como [marathon](/wiki/marathon). La mayoría de los mapas aprobados no tienen dificultades [fácil](/wiki/easy) ni [normal](/wiki/normal).
+
+### Mapas cualificados 
+
+Los mapas cualificados (qualified) vienen indicados con el icono: (![Check icon](/wiki/shared/status/qualified.png)) en la pantalla de selección  de canciones. Estos beatmaps están en la última etapa del proceso de mapeo y modding. Al seleccionarlos, muestran un mensaje que avisa de que:
+
+-No otorgan [PP](/wiki/pp)
```

Comment:
`-No otorgan [PP](/wiki/pp)` -> `- No otorgan [PP](/wiki/pp)`
there is a syntax error: add a space between the hyphen and the text